modules/ismp/clients/parachain/client/src/lib.rs

```diff
@@ -71,7 +72,7 @@ pub mod pallet {
 
 	/// List of parachains that this state machine is interested in.
 	#[pallet::storage]
-	pub type Parachains<T: Config> = StorageMap<_, Identity, u32, ()>;
+	pub type Parachains<T: Config> = StorageMap<_, Identity, u32, ParachainData>;
```

Comment:
You'll need to add a storage migration, since this pallet in used by a couple teams on testnet.